src/shared/ui/Button/ui/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { PropsWithChildren } from "react"
+import { Button, ButtonProps } from "react-bootstrap"
+
+type BasicButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & ButtonProps & PropsWithChildren<unknown>
```

Comment:
IMHO, you won't need React.ButtonHTMLAttributes<HTMLButtonElement> as you already use ButtonProps.

Reply:
Done